src/python/pants/option/global_options.py

```diff
@@ -116,12 +145,39 @@ def from_bootstrap_options(cls, bootstrap_options: OptionValueContainer) -> Exec
             token_header = {"authorization": f"Bearer {oauth_token}"}
             remote_execution_headers.update(token_header)
             remote_store_headers.update(token_header)
+        if bootstrap_options.remote_auth_plugin and (
+            remote_execution or remote_cache_read or remote_cache_write
+        ):
+            auth_plugin_path, auth_plugin_func = bootstrap_options.remote_auth_plugin.split(":")
```

Comment:
This will throw a `ValueError` if the user does not specify PKG:METHOD exactly. Maybe add a better error message for that case (with said error including the expected form of this argument)?